docs/site/FAQ.md

```diff
@@ -47,7 +101,7 @@ reasons:
 - **Improved extensibility** and flexibility. LoopBack 4's core is simpler than
   LoopBack 3.x with well-defined extension points. A lot of responsibility will
   be shifted to extensions (components), which can be JavaScript or TypeScript.
-- Unified tooling. TypeScript developers all use the same IDE: Visual Studio
+- **Unified tooling**. TypeScript developers all use the same IDE: Visual Studio
```

Comment:
While you're here, I'm thinking to modify this sentence. We probably notice there are users not using VSCode. :) Maybe we can change it along the line that there are good proportion of TypeScript developers are using VSCode, so we provide some vscode configuration to make developing LoopBack applications more convenient